FAQUse: add Coinjoin history (#1255)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

![Wasabi Wallet History tab](/History.png "Wasabi Wallet History tab")
:::

:::details
### How can I see coinjoins in the history list?

> A coinjoin is indicated with the shield icon:

Reviewer: ```suggestion
Coinjoin transactions are indicated with a shield icon:
```
